xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -4440,7 +4440,11 @@ bool CVideoPlayer::OnAction(const CAction &action)
     }
   }
 
-  if (dynamic_cast<CDVDInputStreamPVRManager*>(m_pInputStream))
+  //If the current channel is a nullptr, and this is a PVRStream, then this is a saved recording
+  CDVDInputStreamPVRManager* pvrStream = dynamic_cast<CDVDInputStreamPVRManager*>(m_pInputStream);
+  CPVRChannelPtr channel(g_PVRManager.GetCurrentChannel());
+  // If the pvrStream is not a saved recording, then handle the actions below
+  if (pvrStream && channel)
```

Comment:
@hugegreenbug please check for channel only if it results in switching channels. in current state this change kills commerial skipping (edl)